x-pack/test/functional/apps/visualize/hybrid_visualization.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getPageObjects, getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects(['common', 'visualize', 'timePicker']);
+  const inspector = getService('inspector');
+
+  describe('hybrid', () => {
```

Comment:
I think we need something meaningful here to explain suite purpose @LeeDr @rasroh ?